scipy-1.0/paper.tex

```diff
@@ -994,17 +998,18 @@ \subsubsection*{Test suite}
 Deep-shaded area represents lines of code covered by units tests;
 light-shaded area represents lines not covered. With the exception
 of the removal of $\approx 61,000$ lines of compiled code for SciPy
-v0.14, the volume of both compiled and Python
+v0.14, the volume of both compiled (C, C++, and Fortran) and Python
 code has increased between releases, as has the number of lines
-covered by unit tests. SciPy v1.0.0 was composed of $462,574$ lines of
-compiled code with test coverage near $45\%$ and $106,878$ lines of 
-Python code with test coverage near $87\%$.\\
-The reconstitution of historical
-build and test environments was performed using a docker-based approach
-\cite{scipy-cov}. Analysis of lines of source code covered is performed 
-automatically in our continuous integration suite using the \texttt{pytest-cov}
-and \texttt{gcov} libraries. Some decreases in compiled code coverage may reflect
-inclusion of additional vendored code.}
```

Comment:
Happy to add back statement about decreases in compiled code if it's important. Thought it might have been a relic from old versions of the figure in which coverage (%) decrease was more apparent. If it's important, though, maybe put it in the body? Seems like an afterthought here, as it's unrelated to the previous two sentences.

Reply:
Fine to remove, I think